### PR TITLE
services/ticker: scrape asset data (#1080)

### PR DESCRIPTION
**Pull request by accordeiro**
Refs #1080  (Create a function that scrapes stellar.toml data and stores the relevant assets in the db, so we'll later know which market data should be ingested).

### CODE REVIEW

File: services/ticker/internal/scraper/asset_scraper.go [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+	"net/url"
+	"strconv"
+	"strings"
+
+	horizonclient "github.com/stellar/go/exp/clients/horizon"
+	hProtocol "github.com/stellar/go/protocols/horizon"
+)
+
+// nextCursor finds the cursor parameter on the "next" link of an AssetPage
+func nextCursor(assetsPage hProtocol.AssetsPage) (cursor string, err error) {
```
Comment (tomquisel) [inline services/ticker/internal/scraper/asset_scraper.go]: @poliha Alex is the first big user for your new Go SDK 😄! I noticed this utility he wrote, and it seems like a good thing to have in the SDK (if it's not in there already). I really like the JS SDK's `result.next()` feature, something like that would be great.

File: services/ticker/internal/scraper/asset_scraper.go [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+		} else {
+			numTrash++
+		}
+	}
+
+	return
+}
+
+// retrieveAssets retrieves all existing assets from the Horizon API
+func retrievAssets(c *horizonclient.Client) (assets []hProtocol.AssetStat, err error) {
```
Comment (tomquisel) [inline services/ticker/internal/scraper/asset_scraper.go]: retriev -> retrieve

File: services/ticker/internal/scraper/asset_scraper.go (hunk 1 above)
Comment (poliha) [reply services/ticker/internal/scraper/asset_scraper.go]: Yes it's nice to have. It's not yet in the SDK but its on the list of features to add.. There is an issue for this #985

**Comment by poliha** [comment]
All good from me! :ok_hand:

File: services/ticker/internal/scraper/asset_scraper.go (hunk 2 above)
Comment (accordeiro) [reply services/ticker/internal/scraper/asset_scraper.go]: @tomquisel I've just pushed a fix for this typo. Thanks!

File: services/ticker/internal/scraper/asset_scraper.go (hunk 1 above)
Comment (accordeiro) [reply services/ticker/internal/scraper/asset_scraper.go]: Great! Let me know if I can help in any way 😄

**Pull request merged**